app/Console/Commands/AdsharesCrawlCommand.php

```diff
@@ -116,9 +124,13 @@ protected function crawlHosts(Esc $esc, Adselect $adselect)
                 continue;
             }
 
+            $uuids = [];
+
             foreach ($inventory['campaigns'] as $campaign_data) {
                 $campaign_data['source_host'] = $host;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "source_host" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1192)